docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Use ML.NET in a TensorFlow transfer learning scenario for image classification
+description: Discover how to use ML.NET in a TensorFlow transfer learning scenario to classify images by reusing a pre-trained TensorFlow model.
+ms.date: 03/20/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to reuse a pre-trained TensorFlow model with ML.NET so that I can classify images with a small amount of training data.
+---
+# Tutorial: Use ML.NET in a TensorFlow transfer learning scenario for image classification
+
+This sample tutorial illustrates using ML.NET to reuse a pre-trained TensorFlow model to classify images with a small amount of training data via a .NET Core console application using C# in Visual Studio 2017. In the world of machine learning, this is called transfer learning.
```

Comment:
Please also provide at a high level what style of transfer learning are we trying to achieve with this tutorial. 
Retrain penultimate layer?
Retrain wieghts on different layers?
etc.